src/DocTests.jl

```diff
@@ -241,6 +241,7 @@ end
 
 function eval_repl(block, sandbox, meta::Dict, doc::Documenter.Document, page)
     for (input, output) in repl_splitter(block.code)
+        @debug "Evaluating REPL line" input output
```

Comment:
```suggestion
        @debug "Evaluating doctest REPL line" input expected_output = output
```

Reply:
I updated to include this suggestion and the file & line numbers of the doctest.  The output for the same test as above now looks like

```
     Testing Running tests...
     ┌ Debug: Doctesting in temporary directory: /tmp/jl_58qp0Y
     │   modules =
     │    1-element Vector{Module}:
     │     WolframExpr
     └ @ Documenter ~/.julia/dev/Documenter/src/doctest.jl:73
     ┌ Debug: Document: remotes
     │   remote = nothing
     │   remotes = nothing
     └ @ Documenter ~/.julia/dev/Documenter/src/documents.jl:380
     [ Info: SetupBuildDirectory: setting up build directory.
     [ Info: Doctest: running doctests.
     ┌ Debug: Running doctests.
     └ @ Documenter.DocTests ~/.julia/dev/Documenter/src/DocTests.jl:47
     ┌ Debug: Evaluating doctest REPL line from ~/.julia/dev/WolframExpr/src/WolframExpr.jl:20-30
     │   input = "using WolframExpr"
     │   expected_output = ""
     └ @ Documenter.DocTests ~/.julia/dev/Documenter/src/DocTests.jl:246
     ┌ Debug: Verifying doctest at ~/.julia/dev/WolframExpr/src/WolframExpr.jl:20-30
     │
     │ ```jldoctest
     │ julia> using WolframExpr
     │
     │ julia> f = string_to_function("x+y", [:x, :y]);
     │
     │ julia> f(1, 2)
     │ 3
     │
     │ julia> f(1.2, 2)
     │ 3.2
     │ ```
     │
     │ Subexpression:
     │
     │ using WolframExpr
     │
     │ Evaluated output:
     │
     │
     │
     │ Expected output:
     │
     │
     └ @ Documenter.DocTests ~/.julia/dev/Documenter/src/DocTests.jl:354
     ┌ Debug: Evaluating doctest REPL line from ~/.julia/dev/WolframExpr/src/WolframExpr.jl:20-30
     │   input = "f = string_to_function(\"x+y\", [:x, :y]);"
     │   expected_output = ""
     └ @ Documenter.DocTests ~/.julia/dev/Documenter/src/DocTests.jl:246
```